webapp/channels/src/components/timestamp/timestamp.tsx

```diff
@@ -353,7 +355,8 @@ class Timestamp extends PureComponent<Props, State> {
 
                 return {year, month, day};
             },
-            useTime = {hour, minute},
+            displayTimeWithSeconds = false,
```

Comment:
do we want the seconds to be always shows? This component is used in threads, post edited time, custom status @johndavidlugtu

Reply:
Are the components you're mentioning on hover or the timestamp that's readily visible?
Example - thread:
<img width="324" alt="Screenshot 2023-12-13 at 9 43 21 AM" src="https://github.com/mattermost/mattermost/assets/101303489/5885ec58-e29c-40b6-a6c6-570f5f45994f">

Reply:
I think as-coded, it's only showing seconds in the tooltip of `post_time`, which is not used in post edited time, etc. as far as I can tell

Reply:
Sounds good. Having it only on the tooltip is the desired behaviour at this point